.travis.yml

```diff
@@ -14,12 +14,13 @@ env:
   - IMAGE_HASH_TAG=${TRAVIS_COMMIT::12}
 before_install:
 - echo ${DOCKER_PASSWORD} | docker login -u=decaftravis --password-stdin
+- pip install --user cookiecutter
```

Comment:
```suggestion
- pip install cookiecutter
```

Reply:
That errored [here](https://travis-ci.org/DD-DeCaF/wsgi-base/builds/609205649#L191)

Reply:
o.0 okay